### DISCUSSION TITLE
lolMiner Zombie problem

### DISCUSSION THREAD

**Discussion by izhere** [Lolliedieb/lolMiner-releases#307] [Q&A]
I have B450 Aorus Pro motherboard with 3 PCIe slots. I have 3x RX570 4GB Asus Strix. I can't make all running, the first one 24mh/s, the other 2 no more than 8mh/s

How can I fix this. Thank you :)

**Comment by bullet97** [comment]
Oly one slot x16. And this one will be give you good hashrate, for example 24. Another 2 slots is x4 and x1, and that's not enough for good hashrate

**Reply by nitroyouth** [reply]
Not entirely true - I'm getting same hashrates on x4 or x8 (RX470 4G ~20MH/s). But I can confirm izhere's issue - with more than one card the hash rates of second and further cards drop to ~8MH:
Average speed (30s): 22.93mh/s | 8.84mh/s Total: 31.78 mh/s
(tested on Z390UD as well as prehistoric Z77X-D3H)

**Reply by Migel-del** [reply]
You must need to know Russian language. There is too much experimental information on Russian forums, which can help you.

For your problem: B450 Aorus Pro contain only 1 (one) slot which support x16 speed. Other x16 slots work on x4. You can check full info about your MB. And i give 99% that your CPU support only 1 (one) x16 or 2 (who) x8 + x8 speed.

If you want use 2 GPU in zombie i think you need to look on X79 X99 MB and Xeon CPU-s.
